src/controllers/comment.controller.js

```diff
@@ -27,4 +27,22 @@ export default class CommentController {
       return Response.InternalServerError(res, 'Could not delete comment');
     }
   }
+
+  /**
+ * get comments associated with a travel request
+ * @param {ServerRequest} req
+ * @param {ServerResponse} res
+ * @returns {ServerResponse} response
+ */
+  static async getComments(req, res) {
+    try {
+      const { request_id: id } = req.params;
+      const result = await Comment.findAll({ attributes: ['id', 'comment'], where: { request_id: id, delete_status: false } });
+      return Response.Success(res, {
+        ...result
+      }, 200);
+    } catch (error) {
+      return Response.InternalServerError(res, 'Could not populate the comment(s)');
```

Comment:
**Nothing Too Important** :  When it comes to server errors, it's nice to just send a vague message like `Internal server error` and nothing else. This will ensure you do not reveal the vulnerabilities of your app.

Reply:
Hi, @O-Osahon Thanks for the  review, i personally think is good to send a descriptive message to your end user when something did not work well, eg if an app needs internet connection to work well and the user's internet is not on, it is more user friendly to tell the user to `check his/her internet connection` than to display a vague message like `server error`, i dont see that as exposing the weakness of the app as the absence of internet connection on the users end does not make the app vulnerable. i have equally gone through some of our team mates codes and they follow the same pattern of sending a user friendly message when things don't go the way they should.  Attached below are some samples from the remote develop branch. 
![Screenshot (473)](https://user-images.githubusercontent.com/33726993/64322820-91035d80-cfc3-11e9-9966-cf18d8966645.png)
![Screenshot (474)](https://user-images.githubusercontent.com/33726993/64322823-906ac700-cfc3-11e9-8709-33b5162c5b46.png)
![Screenshot (475)](https://user-images.githubusercontent.com/33726993/64322824-919bf400-cfc3-11e9-8aa9-52ad14719de0.png)
![Screenshot (476)](https://user-images.githubusercontent.com/33726993/64322825-919bf400-cfc3-11e9-88ae-0e1d4c1cabe2.png)

Reply:
I understand what you're saying. However, that holds true when the error is from the client (e.g wrong input or authentication error). A 500 error means that something is wrong from the developer's end. In this case it is not ideal to let the client know what problem the server-side code has. Giving users details of a server error could lead to security issues. Again, this is not very explicit, but we could use a single generic message for 500 errors. 

Like I earlier said it's not worth splitting hairs over. If you insist, I'll resolve the comment

Reply:
@O-Osahon 
Ok, 
Please don't misunderstand my response, i appreciate your review. thanks so much

Reply:
@jiokeokwuosa You're right. No user would come back to an application that cannot point them in the correct line when an error happens. Solving an error that should normally take seconds, will then take years, if left to guess work. Imagine trying to create a new Github account, and for some reasons, you were using an email that already existed; or perhaps, trying to watch a movie on Netflix with poor internet and you get internal server error; when it could have warned you the email already exists or your internet is poor in both scenarios. I think what @O-Osahon tried to imply is `not displaying your stack traces` to the end-users; but logging them instead. But that's a different ball-game.

Reply:
Very well then. I guess I've been handling 500 errors wrongly all along.